app/src/ui/index.tsx

```diff
@@ -150,12 +150,14 @@ dispatcher.setAppFocusState(remote.getCurrentWindow().isFocused())
 
 ipcRenderer.on('focus', () => {
   const state = appStore.getState().selectedState
```

Comment:
while this isn't part of your changed lines, how do you feel about us renaming `state` to `selectedState`, ex:
```
  const { selectedState } = appStore.getState()

  // Refresh the currently selected repository on focus (if
  // we have a selected repository).
  if (selectedState && selectedState.type === SelectionType.Repository) {
    dispatcher.refreshRepository(selectedState.repository)
  }
```
i'd prefer to call it selectedState just so we don't mistake it for having all the state info

Reply:
I love it